docs/csharp/quick-starts/list-collection.yml

```diff
@@ -0,0 +1,162 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Collections in C#
+metadata:
+  title:  Collections in C#. Learn to use sequences and collections in C#.
+  description: In this tutorial, you'll use your browser to learn C# interactively. You write C# code and see the results of compiling and running your code directly in the browser.
+  audience: Developer
+  level: Beginner
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that create, modify and explore collections and arrays.
+- title: Create lists
+  durationInMinutes: 2
+  content: |
+    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the **Run** button:
+
+    ```csharp
+    var names = new List<string> { "<name>", "Ana", "Felipe" };
+    foreach (var name in names)
+    {
+      Console.WriteLine($"Hello {name.ToUpper()}!");
+    }
+    ```
+
+    You've just created a list of strings, added three names to that list, and printed out the names in all CAPS. You're using concepts
+    that you've learned in earlier quick starts to loop through the list.
+
+    Another new feature here is **interpolated strings**.  When you precede a `string` with the `$` character, you can embed C# code in the string declaration. The actual string replaces that C# code with the value it generates. In this example, it replaces the `{name.ToUpper()}` with each name, converted to capital letters, because you called the `ToUpper` method.
+
+    Let's keep exploring.
+    
+    > [!NOTE]
+    > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
+
+- title: Modify list contents
```

Comment:
I think we should update the dot.net In-Browser Tutorial to start here, once this is live. Sound right to you?

Reply:
Yes. You'll have an anchor tag for `#modify-list-contents`. See the latest update for the note at the top of this file.